deps/3rdparty/tests/libDispatch/libDispatch.UnitTest.vcxproj

```diff
@@ -232,5 +223,5 @@
     <ClInclude Include="dispatch_test.h" />
   </ItemGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
-
+  <Import Project="$(MSBuildThisFileDirectory)\..\..\..\..\common\winobjc.packagereference.override.targets" Condition="Exists('$(MSBuildThisFileDirectory)\..\..\..\..\common\winobjc.packagereference.override.targets')" />
```

Comment:
can we shorten these with StarboardBasePath or something?

Reply:
Doesn't seem to be shorter, but does seem more accurate to do.